libs/design-system/src/components/ThemeRadio.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
 
 export function ThemeRadio({ className, tooltipClassName, tabIndex }: Props) {
   const { theme, setTheme } = useTheme()
+
```

Comment:
Relating to your other comment, to avoid the layout shift you could render the whole thing with a fixed value in fallback so that it looks the same/takes the same exact space, and also does not hydration mismatch.
```
return <ClientSideOnly fallback={<ThemeRadioInner value={false} ... />}>
  <ThemeRadioComponent value={theme} ... />
  </ClientSideOnly>
  ```